app/api/vector.py

```
@@ -0,0 +1,70 @@
+from app.database.connection import get_db
+from app.models import Store, User, UsageHistory, Category, Brand, StoreEmbedding, BrandEmbedding
+from typing import List
+from datetime import datetime
+
+router = APIRouter()
+
+model = SentenceTransformer("sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2")
+
+@router.post("/vectors/store")
+def generate_store_vectors(db: Session = Depends(get_db)):
+    stores = db.query(Store).join(Store.brand).filter(Store.brand.has(Brand.description != None)).all()
```

Ruff (0.12.2)

Comparison to `None` should be `cond is not None`

Replace with `cond is not None`

(E711)